src/modules/settings/ChatbotSettings.tsx

```diff
@@ -275,6 +282,33 @@ const ChatbotSettings = (): JSX.Element => {
               </Accordion>
             </Stack>
           </Box>
+          <Box>
+            <Stack>
+              <FormLabel>{t('MODEL_VERSION')}</FormLabel>
+              <Typography variant="caption" color="text.secondary">
+                {t('CHATBOT_MODEL_VERSION_HELPER')}
+              </Typography>
+            </Stack>
+            <Select
+              value={newChatbotVersion}
+              label={t('MODEL_VERSION')}
+              onChange={(event) => {
+                setNewChatbotVersion(event.target.value as GPTVersion);
+                setUnsavedChanges(true);
+              }}
+              fullWidth
+              renderValue={(selected) => <Typography>{selected}</Typography>}
+            >
+              {Object.entries(GPTVersion).map(([key, v]) => (
+                <MenuItem key={v} value={v} sx={{ display: 'block' }}>
+                  <Typography>{v}</Typography>
```

Comment:
I really like the display sou have done with the explanations. Would it be possible to add a chip next to the model name that say "Recommended" (for 3.5-turbo only)  ?